src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Theme/Blocks/logo_configuration.html.twig

```diff
@@ -25,102 +25,129 @@
+          <p class="logo-card-description description-in-tab">
+            {{ 'Will appear on email headers. If undefined, the header logo will be used.'|trans({}, 'Admin.Design.Help') }}
+          </p>
 
-      <div class="form-group">
-        <div class="form-control-label">
-          {{ ps.label_with_help(('Invoice logo'|trans({}, 'Admin.Design.Feature')), ('Will appear on invoice headers.'|trans({}, 'Admin.Design.Help')) ~ ('Warning: you can use a PNG file for transparency, but it can take up to 1 second per page for processing. Please consider using JPG instead.'|trans({}, 'Admin.Design.Help'))) }}
+          <div class="text-center logo-image-container">
+            <img
+              class="email-logo"
+              src="{{ mailLogoPath }}"
+              alt="{{ 'Mail logo'|trans({}, 'Admin.Design.Feature') }}"
+              title="{{ 'Mail logo'|trans({}, 'Admin.Design.Feature') }}"
+            >
+          </div>
+          <div class="input-in-tab">
+            {{ ps.form_group_row(shopLogosForm.options.mail_logo, {}, {'help': headerLogoPath == mailLogoPath ? 'Warning: if no email logo is available, the main logo will be used instead.'|trans({}, 'Admin.Design.Notification') : '' }) }}
+          </div>
+        </div>
+        <div class="tab-pane" id="invoice-configuration" role="tabpanel" aria-labelledby="logo-tab">
+          <p class="logo-card-description description-in-tab">
+            {{ 'Will appear on invoice headers.'|trans({}, 'Admin.Design.Help') }}
+          </p>
 
-          <img src="{{ invoiceLogoPath }}" alt="{{ 'Invoice logo'|trans({}, 'Admin.Design.Feature') }}" title="{{ 'Invoice logo'|trans({}, 'Admin.Design.Feature') }}">
+          <div class="text-center logo-image-container">
+            <img
+              class="invoice-logo"
+              src="{{ invoiceLogoPath }}"
+              alt="{{ 'Invoice logo'|trans({}, 'Admin.Design.Feature') }}"
+              title="{{ 'Invoice logo'|trans({}, 'Admin.Design.Feature') }}"
+            >
+          </div>
+          <div class="input-in-tab">
+            {{ ps.form_group_row(shopLogosForm.options.invoice_logo, {}, {'help': headerLogoPath == invoiceLogoPath ? 'Warning: if no invoice logo is available, the main logo will be used instead.'|trans({}, 'Admin.Design.Help') : '' }) }}
+          </div>
         </div>
+      </div>
+    {% endblock %}
+  {% endembed %}
 
-        {{ form_widget(shopLogosForm.invoice_logo) }}
+  {% embed '@PrestaShop/Admin/Improve/Design/Theme/Blocks/Partials/logo_card.html.twig' %}
+    {% block card_title %}
+      {{ form_widget(shopLogosForm.options.favicon_is_restricted_to_shop) }} {{ 'Favicon'|trans({}, 'Admin.Design.Feature') }}
+    {% endblock %}
+    {% block card_content %}
+      <p class="logo-card-description">
+        {{ 'It is the small icon that appears in browser tabs, next to the title'|trans({}, 'Admin.Design.Help') }}
```

Comment:
Wording has been updated in 1.7.4, the current tooltip is: _It is the small icon that appears in browser tabs, next to the title._